### PR TITLE
[t7] Added quiz start page

### PR DESCRIPTION
**Pull request by harshithad0703**

### CODE REVIEW

File: src/components/StartQuizPage/index.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Button, Form, Input } from 'antd';
+
+import  './index.css';
+
+const QuizStart = () => {
```
Comment (faraazb) [inline src/components/StartQuizPage/index.jsx]: QuizStart can be StartQuizPage?

File: src/components/StartQuizPage/index.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    return (
+        <div className="quizStart" >
+            <Form id='form'>
+                <Form.Item
+                    name="username"
+                    rules={[{
+                        required: true,
+                        message: 'Please enter your username!',
```
Comment (faraazb) [inline src/components/StartQuizPage/index.jsx]: If the username doesn't match the pattern, it still shows the same message, so we can change the message to "Please enter a valid username"

File: src/components/StartQuizPage/index.jsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+                        pattern: '^[a-zA-Z]+( [a-zA-Z]+)?$'
```
Comment (faraazb) [inline src/components/StartQuizPage/index.jsx]: Even if you remove everything after the first plus sign, the regex is the same right? Also, I think underscore and dot should be allowed. Maybe this `^[a-zA-Z._]+$`

**Review by faraazb — CHANGES_REQUESTED** [review]

File: src/components/index.js [hunk 4]
```diff
@@ -1,6 +1,8 @@
 import Header from "./Header/Header";
+import QuizStart from "./StartQuizPage/index.jsx";
```
Comment (faraazb) [inline src/components/index.js:2]: We won't need this `index.js` file, you can delete it. We will import from "./components/componentName" like Vaibhav and Kashmeera are doing

File: src/components/StartQuizPage/index.jsx (hunk 2 above)
Comment (harshithad0703) [reply src/components/StartQuizPage/index.jsx]: yeah I'll update the changes

**Review by faraazb — APPROVED** [review]
Approved

**Pull request merged**